### PR TITLE
feat: support to custom notification type

### PR DESCRIPTION
**Pull request by froucher**
https://github.com/amtrack/force-dev-tool/issues/211

### CODE REVIEW

**Comment by froucher** [comment]
Hello,

I have successfully create a local test with a CustomNotificationType, create a delta, and deployed with ant the delta folder. Please @amtrack let me know if you need something more to validate this change.

Thanks!

File: lib/describe-metadata-result.json [hunk 1]
```diff
@@ -685,7 +685,13 @@
 		"metaFile": false,
 		"suffix": "lightningExperienceThemes",
 		"xmlName": "LightningExperienceTheme"
-	}],
+	}, {
+		"directoryName": "notificationtypes",
+		"inFolder": false,
+		"metaFile": false,
+		"suffix": "notificationtypes",
```
Comment (amtrack) [inline lib/describe-metadata-result.json]: Shouldn't this be `"suffix": "notiftype"`?
`suffix` is the file extension and looking at any of my sfdx orgs in `.sfdx/USERNAME/metadataTypeInfos.json` i see the following:

```
        "CustomNotificationType": {
            "metadataName": "CustomNotificationType",
            "isAddressable": true,
            "isSourceTracked": true,
            "ext": "notiftype",
            "hasContent": false,
            "defaultDirectory": "notificationtypes",
            "nameForMsgs": "Custom Notification Type",
            "nameForMsgsPlural": "Custom Notification Types",
            "contentIsBinary": false,
            "hasStandardMembers": false,
            "deleteSupported": true,
            "decompositionConfig": {
                "strategy": "nonDecomposed",
                "workspaceStrategy": "nonDecomposed",
                "commitStrategy": "fineGrainTracking",
                "metadataName": "CustomNotificationType",
                "useSparseComposition": false,
                "decompositions": [],
                "contentStrategy": "nonDecomposedContent"
            },
            "hasVirtualSubtypes": false
        },
```

File: lib/describe-metadata-result.json [hunk 2]
```diff
@@ -685,7 +685,13 @@
 		"metaFile": false,
 		"suffix": "lightningExperienceThemes",
```
Comment (amtrack) [inline lib/describe-metadata-result.json]: and by the way: I think this should then be: `"suffix": "lightningExperienceTheme"` (singular)

**Comment by amtrack** [comment]
@froucher Thanks for your PR. Please look at my comments.

**Comment by froucher** [comment]
Ok, thanks for your comments, then I will test both suffix changes and update this thread with the result.

**Comment by froucher** [comment]
Hi @amtrack,

I have fixed suffix of customNotificationTypes, lightningExperienceThemes and BrandingSets, and tested again if the delta is well created for the three metadata, and it seems all is working right.

Please let me know if I must test something more.

Thanks!

**Comment by froucher** [comment]
Hi,

Just for clarification, these are examples of suffixes:

force-app/main/default/brandingSets/LEXTHEMINGThemeTest.brandingSet-meta.xml
force-app/main/default/lightningExperienceThemes/ThemeTest.lightningExperienceTheme-meta.xml
force-app/main/default/notificationtypes/forcedevtool_custom_notification_test.notiftype-meta.xml

src/lightningExperienceThemes/ThemeTest.lightningExperienceTheme
src/notificationtypes/forcedevtool_custom_notification_test.notiftype
src/brandingSets/LEXTHEMINGThemeTest.brandingSet

Regards

**Comment by amtrack** [comment]
@froucher The changes look good!
Would it be possible for you to cleanup this PR to only contain two commits with the following commit messages?

- feat: add support for CustomNotificationType in changeset command (#211)
- fix: revise file extensions for BrandingSet and LightningExperienceTheme

I would like the release notes to be generated nice and clearly ;-)

**Comment by froucher** [comment]
Sure, I have created a new clean Pull Request: https://github.com/amtrack/force-dev-tool/pull/213

**Comment by amtrack** [comment]
closing in favor of #213

**Pull request closed without merging**